gcs/CONFIGURATION.md

```diff
@@ -37,6 +37,15 @@
 
     Delegation Token binding class.
 
+*   `fs.gs.delegation.token.instantiation.strategy` (default `INSTANCE_PER_SERVICE`)
```

Comment:
nit: add colon after `default` for consistency with rest of the values: `default:`